common/functions-v2/tests/it/scalars/string.rs

```diff
@@ -34,6 +34,9 @@ fn test_string() {
     test_char_length(file);
     test_to_base64(file);
     test_from_base64(file);
+    test_quote(file);
+    test_reverse(file);
+    test_ascii(file);
```

Comment:
It will be better if we add more complex cases like emojis and Chinese.

Reply:
The `reverse` function simply reverse the byte array now, so I think `Chinese` and `emojis` tests are no meaning.

Shall we treat the input as utf8 string, then reverse?

Reply:
I think it is meaningful to record the behavior even if it's not useful to the user. 

Besides, what's the plan on the topic of utf8 char reserve? @sundy-li

Reply:
> Besides, what's the plan on the topic of utf8 char reserve

I  think we can have another function name for that, eg: `reverse_utf8`

Reply:
> I think it is meaningful to record the behavior even if it's not useful to the user.

Added.